corehq/apps/es/es_query.py

```diff
@@ -405,6 +428,14 @@ def facets(self):
         results = namedtuple('facet_results', [f.name for f in facets])
         return results(**{f.name: f.parse_result(raw) for f in facets})
 
+    @property
+    @memoized
+    def aggregations(self):
+        aggregations = self.query._aggregations
+        raw = self.raw.get('aggregations', {})
+        results = namedtuple('aggregation_results', [a.name for a in aggregations])
+        return results(**{a.name: a.parse_result(raw) for a in aggregations})
```

Comment:
We should definitely replace facets usage with aggregations (facets are removed in 2.0) but I don't think we can just use the same API (though we could probably just use aggregations behind the scenes for the current facet usage). Aggregations are much more powerful & complex since they can be nested.